src/site/content/en/animations/animations-guide/index.md

```diff
@@ -1,45 +1,45 @@
 ---
 layout: post
-title: Creating high performance animations
+title: How to create high-performance CSS animations
 authors:
   - rachelandrew
+  - kaycebasques
 description: |
-  In this guide you will learn how to create high performance CSS animations.
-date: 2020-10-02
+  Use the transform and opacity CSS properties as much as possible, and avoid anything that
+  triggers layout or painting.
+date: 2020-10-06
 tags:
+  - animations
   - performance
 ---
 
-In the overview to CSS animations you learned how the performance of CSS animations is closely tied
-to which stage of the rendering pipeline the animation triggers.
-In this guide you will learn more about how to create high performance animations,
-and the tools that are available in browsers to help.
+This guide teaches you how to create high-performance CSS animations.
 
-## Rendering pipeline triggers
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations.
 
-You should try to restrict animations to changes to `translate`, `scale`,
-and `rotation` using the [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property,
-and [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity).
-These things can be animated relatively cheaply due to the fact they act only on the compositing stage of the rendering pipeline.
-See [Why are some animations slow?](/animations-overview) for more information about the pipeline.
+## Browser compatibility
 
-Animating other properties will trigger earlier stages of the pipeline.
-The stage that is triggered is not always the same across browsers.
-If you need to animate a property other than `transform` or `opacity`
-the [CSS Triggers](https://csstriggers.com/) site can help you understand where in the pipeline the process will be triggered.
+All of the CSS properties that this guide recommends have good cross-browser support.
 
-Your browser DevTools will also help you to see how animations impact rendering,
-as you can see below.
+* [`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform#Browser_compatibility)
+* [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#Browser_compatibility)
+* [`will-change`](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change#Browser_compatibility)
 
-## Triggering layout
+## Move an element {: #move }
 
-As layout is the earliest stage of the pipeline,
-you should avoid triggering the process here if possible.
-Quite often an expensive operation using a property that will trigger layout can be changed to one which does not.
+To move an element, use the `translate` or `rotation` keyword values of the
+[`transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/transform) property.
```

Comment:
We may want to recap the conceptual content (as you originally did) but we should lead off the section with the instructions on how to do the task

Reply:
I think then we should move that before and after comparison down to the DevTools bit, as it's a bit jumbled now. Personally I'd put it back in the original piece (the overview) which is where it came from, as I think a worked example explains the theory. I moved it to this part because of your initial comments.

If it can't go in the overview then let's move it down to the stuff about diagnosing low performance animations, so it becomes more about "if you see this, then change it to this".